src/content/docs/dropins/cart/initialization.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Cart initialization
+description: Learn how to configure initializer for the cart drop-in component.
+---
+
+import OptionsTable from '@components/OptionsTable.astro';
+
+The cart drop-in component initializer provides options for configuring language definitions and extending the default models with new fields and transformers.
+
+## Configuration options
+
+The cart component initializer accepts the following configuration options:
+
+<OptionsTable
+  options={[
+    ['Option', 'Type', 'Req?', 'Description'],
+    ['disableGuestCart', 'boolean', 'No', 'Whether guest carts have been disabled.'],
+    ['langDefinitions', 'object', 'No', 'Provides language definitions for internationalization (i18n).'],
+    ['models', 'object', 'No', 'Extend the default models (CartModel, CustomerModel, and OrderModel) with new fields and transform them as needed.'],
```

Comment:
Does it accept anything except `CartModel`? On line 63, we say `CartModel` only.